src/Console/EditCommand.php

```diff
@@ -0,0 +1,265 @@
+<?php
+/**
+ * This file is part of byrokrat\giroapp.
+ *
+ * byrokrat\giroapp is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * byrokrat\giroapp is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with byrokrat\giroapp. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2016-17 Hannes Forsgård
+ */
+
+declare(strict_types = 1);
+
+namespace byrokrat\giroapp\Console;
+
+use byrokrat\giroapp\Mapper\DonorMapper;
+use byrokrat\giroapp\Builder\DonorBuilder;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Console\Question\Question;
+use byrokrat\giroapp\Events;
+use byrokrat\banking\AccountFactory;
+use byrokrat\id\IdFactory;
+use byrokrat\amount\Currency\SEK;
+use byrokrat\giroapp\Model\Donor;
+use byrokrat\giroapp\Model\PostalAddress;
+use byrokrat\giroapp\Event\DonorEvent;
+use byrokrat\giroapp\Builder\MandateKeyBuilder;
+
+/**
+ * Command to edit an existing mandate
+ */
+class EditCommand implements CommandInterface
+{
+    /**
+     * @var Command
+     */
+    private $command;
+
+    public function configure(Command $command)
+    {
+        $this->command = $command;
+        $command->setName('edit');
+        $command->setDescription('Edit an existing donor');
+        $command->setHelp('Edit a donor in the database.');
+        $command->addArgument('donor-key', InputArgument::REQUIRED, 'Donor key or payernumber');
+        $command->addOption(
+            'use-payernumber',
```

Comment:
Agree. I actually had force-mandate-key or somthing like that in an earlier version....
But yes, is better, and is a good place to be picky.